ci-operator/config/konflux-ci/e2e-tests/konflux-ci-e2e-tests-main.yaml

```
@@ -128,6 +128,14 @@ tests:
       SCENARIO: max-concurrency-advanced
     workflow: redhat-appstudio-load-test-max-concurrency
   timeout: 8h0m0s
+- always_run: false
+  as: max-concurrency-basic-increase-20u
+  steps:
+    cluster_profile: aws-rhtap-performance
+    env:
+      SCENARIO: load-test-scenario.max-concurrency-basic.increase-20u
```

@rh-rahulshetty the SCENARIO should be without the `load-test-scenario.` prefix.
```suggestion
      SCENARIO: max-concurrency-basic.increase-20u
```

---

Thank you for pointing out! Updated **SCENARIO** to *max-concurrency-basic.increase-20u*